source-academy/frontend
# Grading Overview - Clickable Horizontal Scrollbar

### Description

Resolves #2816.

The fix is a little hacky as it's not really possible to add a ```style``` or ```className``` to the Tremor UI's ```Footer``` component to style it. Due to how ```Footer``` is implemented in Tremor UI, where it sticks to the bottom of the parent element, and blocks the scrollbar, it's basically impossible (at least to my knowledge) to make the scrollbar clickable without directly/indirectly styling the ```Footer```. However, if we update Tremor UI to the later versions, they do support adding ```style``` and ```className```, which would allow the fix to be less hacky.

As for the commit history, just note that only commits from [7a8849d](https://github.com/source-academy/frontend/pull/2824/commits/7a8849ddeac8fa4d7be7bdde4d01ca5585a015c7) are relevant, the rest before have been merged.



### Type of change


### How to test
- Open Grading page
- Make sure there's some things in the table
- Make your window horizontally smaller if it's too large
- Click and drag the scrollbar



### Checklist
